tests/e2e/util.go

```diff
@@ -6119,8 +6127,15 @@ func writeDataOnRawBlockVolume(ns string, podName string, devicePath string, tes
 	_, err = framework.RunKubectl(ns, cmd...)
 	framework.ExpectNoError(err, fmt.Sprintf("failed to write testdata inside the pod: %q", podName))
 
-	cmd = []string{"exec", podName, "--namespace=" + ns, "--", "/bin/sh", "-c",
-		fmt.Sprintf("/bin/dd if=/tmp/data_to_write of=%v", devicePath)}
+	// If dataSize is given, fill the device with random data of size given.
+	// Otherwise write the testData given.
+	if dataSizeInMB != 0 {
+		cmd = []string{"exec", podName, "--namespace=" + ns, "--", "/bin/sh", "-c",
+			fmt.Sprintf("/bin/dd if=/dev/urandom of=%v bs=%vM count=1 conv=fsync", devicePath, dataSizeInMB)}
```

Comment:
`bs` is block size for each IO call, if you pass values in TB for `dataSizeInMB` system may not be able to handle it. Generally used block sizes are 512, 4k, 64k, 1m, 4m. I would recommend you to use 1m.

Reply:
done